src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -242,46 +233,11 @@ private boolean isLoaded(File fileToOpen) {
     }
 
     private void setLookAndFeel() {
-        try {
-
-            if (OS.WINDOWS) {
-                UIManager.setLookAndFeel(WINDOWS_LOOK_AND_FEEL);
-            }
-            if (OS.OS_X) {
-                UIManager.setLookAndFeel(OSX_AQUA_LOOk_AND_FEEL);
-            } else {
-                UIManager.setLookAndFeel(NIMBUS_LOOK_AND_FEEL);
-            }
-            // On Linux, Java FX fonts look blurry per default. This can be improved by using a non-default rendering
-            // setting. See https://github.com/woky/javafx-hates-linux
-            if (Globals.prefs.getBoolean(JabRefPreferences.FX_FONT_RENDERING_TWEAK)) {
-                System.setProperty("prism.text", "t2k");
-                System.setProperty("prism.lcdtext", "true");
-            }
-        } catch (UnsupportedLookAndFeelException | ClassNotFoundException | InstantiationException | IllegalAccessException e) {
-            try {
-                LOGGER.warn("Setting Look and Feel to Nimbus", e);
-
-                UIManager.setLookAndFeel(NIMBUS_LOOK_AND_FEEL);
-            } catch (Exception ex) {
-                LOGGER.warn("Look and feel could not be set", e);
-            }
-
-        }
-
-        // In JabRef v2.8, we did it only on NON-Mac. Now, we try on all platforms
-        boolean overrideDefaultFonts = Globals.prefs.getBoolean(JabRefPreferences.OVERRIDE_DEFAULT_FONTS);
-        if (overrideDefaultFonts) {
-            int fontSize = Globals.prefs.getInt(JabRefPreferences.MENU_FONT_SIZE);
-            UIDefaults defaults = UIManager.getDefaults();
-            Enumeration<Object> keys = defaults.keys();
-            for (Object key : Collections.list(keys)) {
-                if ((key instanceof String) && ((String) key).endsWith(".font")) {
-                    Font font = (Font) UIManager.get(key);
-                    font = new FontUIResource(font.getName(), font.getStyle(), fontSize);
-                    defaults.put(key, font);
-                }
-            }
+        // On Linux, Java FX fonts look blurry per default. This can be improved by using a non-default rendering
+        // setting. See https://github.com/woky/javafx-hates-linux
+        if (Globals.prefs.getBoolean(JabRefPreferences.FX_FONT_RENDERING_TWEAK)) {
```

Comment:
Is there a Linux switch anywhere?

Reply:
Yes, it is turned on by default only on Linux: `defaults.put(FX_FONT_RENDERING_TWEAK, OS.LINUX);`